client/src/components/DataFiles/DataFilesToolbar/DataFilesToolbar.jsx

```diff
@@ -137,10 +142,18 @@ const DataFilesToolbar = ({ scheme, api }) => {
     const filteredSelected = selectedFiles.filter(
       (f) => status[f.system + f.path] !== 'SUCCESS'
     );
+
+    const system = selectedFiles[0].system;
+
+    const selectedSystem = systems.find(
+      (s) => s.system === system && s.scheme === scheme
+    );
+
     dispatch({
       type: 'DATA_FILES_TRASH',
       payload: {
         src: filteredSelected,
+        homeDir: selectedSystem?.homeDir || '',
         reloadCallback: reloadPage,
       },
     });
```

Comment:
The [empty](https://github.com/TACC/Core-Portal/pull/769/files#diff-6470a2f7efa2fe62ab04c44c388ccebd79e5da19665eb21fd75ee753031b8f92R162) function needs to be configured for v3 as well. Currently fails to empty the trash
![Screen Shot 2023-03-07 at 2 17 13 PM](https://user-images.githubusercontent.com/20326896/223542882-68e4fef9-7cb9-4350-a0e0-cfc3518e5f5d.png)